tests/helpers/main.ts

```diff
@@ -157,9 +162,19 @@ export const importFunctionFile = async function <T = any>(functionPath: string)
   return result.default === undefined ? result : result.default
 }
 
-export const normalizeFiles = function (fixtureDir: string, { name, mainFile, runtime, extension, srcFile, schedule }) {
-  const mainFileA = normalize(`${fixtureDir}/${mainFile}`)
-  const srcFileA = srcFile === undefined ? {} : { srcFile: normalize(`${fixtureDir}/${srcFile}`) }
-
-  return { name, mainFile: mainFileA, runtime, extension, schedule, ...srcFileA }
+export const normalizeFiles = function (
+  fixtureDir: string,
+  {
+    mainFile,
+    srcFile,
+    ...rest
+  }: ListedFunction & {
+    srcFile?: string
+  },
+) {
+  return {
+    mainFile: relative(fixtureDir, mainFile),
+    srcFile: srcFile ? relative(fixtureDir, srcFile) : undefined,
+    ...rest,
+  }
```

Comment:
This changed instead of creating an absolute path to creating a relative path for the snapshots. This relates to the changes in `list_functions(_files).test.ts`